src/System.IO.FileSystem/src/project.json

```diff
@@ -23,14 +23,14 @@
         "dotnet5.4"
       ]
     },
-    "netstandard1.7": {
+    "netstandard1.3": {
       "dependencies": {
         "Microsoft.NETCore.Platforms": "1.0.1",
         "System.Collections": "4.0.10",
         "System.Diagnostics.Contracts": "4.0.0",
         "System.Diagnostics.Debug": "4.0.10",
         "System.Diagnostics.Tools": "4.0.0",
-        "System.IO": "4.0.10",
+        "System.IO": "4.2.0-beta-devapi-24407-01",
```

Comment:
Might simply be an artifact of accidentally switching the order.  I'll try and revert.